### PR TITLE
LCAM-240- CC outcome

### PR DESCRIPTION
**Pull request by loganathanMOJ**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/LCAM-240)

Describe what you did and why.

## Checklist

Before you ask people to review this PR:

- [ ] Tests should be passing: `./gradlew test`
- [ ] GitHub should not be reporting conflicts; you should have recently run `git rebase main`.
- [ ] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [ ] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- [ ] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: crown-court-proceeding/src/test/resources/application.yaml [hunk 1]
```diff
@@ -57,6 +57,8 @@ maatApi:
   rep-order-endpoints:
     find-url: ${maatApi.assessments-domain}/rep-orders/{repId}
     update-url: ${maatApi.assessments-domain}/rep-orders
+  rep-order-CCOutcome-endpoints:
```
Comment (muthusubra) [inline crown-court-proceeding/src/test/resources/application.yaml:60]: Please rename to all small case

File: crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/dto/CCOutcomeDTO.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class CCOutcomeDTO {
+
+    private String outCome;
```
Comment (muthusubra) [inline crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/dto/CCOutcomeDTO.java:16]: Please change the case

File: crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/service/ProceedingService.java [hunk 3]
```diff
@@ -54,4 +59,20 @@ public Object graphQLQuery() throws IOException {
         return obj;
     }
 
+    public List<CCOutcomeDTO> getCCOutcome(Integer repId, String laaTransactionId) {
+        List<RepOrderCCOutcomeDTO> repOrderCCOutcomeDTOS = maatCourtDataService.getRepOrderCCOutcomeByRepId(repId, laaTransactionId);
+        List<CCOutcomeDTO> outcomeDTOS = new ArrayList<>();
```
Comment (muthusubra) [inline crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/service/ProceedingService.java:64]: Please change the variable name

File: crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/dto/CCOutcomeDTO.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class CCOutcomeDTO {
```
Comment (muthusubra) [inline crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/dto/CCOutcomeDTO.java:14]: See if you could use just one DTO

File: crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/data/builder/TestModelDataBuilder.java [hunk 5]
```diff
@@ -157,4 +158,12 @@ public static ApiUpdateApplicationRequest getApiUpdateApplicationRequest(boolean
                 .withPassportAssessment(getPassportAssessment());
     }
 
+    public static RepOrderCCOutcomeDTO getRepOrderCCOutcomeDTO(Integer outcomeId, String outCome, LocalDateTime outcomeDate) {
```
Comment (muthusubra) [inline crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/data/builder/TestModelDataBuilder.java:161]: outcome

File: crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/service/MaatCourtDataServiceTest.java [hunk 6]
```diff
@@ -54,4 +57,18 @@ void givenUpdateRepOrderRequest_whenUpdateRepOrderIsInvoked_thenResponseIsReturn
...
     }
+
+    @Test
+    void givenAValidRepId_whenGetRepOrderCCOutcomeByRepIdIsInvoked_thenReturnOutcome() {
+        maatCourtDataService.getRepOrderCCOutcomeByRepId(TestModelDataBuilder.TEST_REP_ID, LAA_TRANSACTION_ID);
+        verify(maatCourtDataClient,atLeastOnce()).getApiResponseViaGET(any(), anyString(), anyMap(), any());
+    }
+
+    @Test
+    void givenAValidRepId_whenGetRepOrderCCOutcomeByRepIdIsInvoked_thenReturnOutcome1() {
```
Comment (muthusubra) [inline crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/service/MaatCourtDataServiceTest.java:68]: Outcome1?

File: crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/dto/CCOutcomeDTO.java (hunk 4 above)
Comment (loganathanMOJ) [reply crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/dto/CCOutcomeDTO.java]: Removed CCOutcomeDTO

**Review by muthusubra — APPROVED** [review]

**Review by muthusubra — APPROVED** [review]

**Pull request merged**